### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by jorikvankemenade**
When using the installation instructions for the automation tools I ran into some small access related issues. Besides this I also added some small clarifications for docker users. 

For the instructions I assumed that most of the access rights should stay with the administrator. A reasonable alternative would be to make the archivematica user the owner of all automation-tools. I don't know what you guys think is best?

Fixes: archivematica/issues#729

### CODE REVIEW

**Comment by jorikvankemenade** [comment]
@ross-spencer could you please review this?

**Comment by ross-spencer** [comment]
Sure thing @jorikvankemenade will try and get to this later today!

File: README.md [hunk 1]
```diff
@@ -53,113 +53,132 @@ Follow each of the steps below to install automation-tools:
...
-/etc/archivematica/automation-tools/`
+    ```
+    cp /usr/lib/archivematica/automation-tools/etc/transfer-script.sh /etc/archivematica/automation-tools/
+    ```
 
 3. Create a transfer source for the automation-tools in the storage service:
 
-You can use an existing transfer source, or set up a new one.
+    You can use an existing transfer source, or set up a new one. For Docker 
+    environments the standard transfer source is `$HOME/.am/ss-location-data/`.
```
Comment (ross-spencer) [inline README.md]: We could highlight that if folks are running docker as `sudo` they might also need to modify the ownership of this folder.

File: README.md [hunk 2]
```diff
@@ -53,113 +53,132 @@ Follow each of the steps below to install automation-tools:
...
-Suggested deployment is to use cron to run a shell script that runs the automate
-transfer tool. Example shell script (for example in
+Suggested deployment is to use cron to run a shell script that runs the 
+automation transfer tool. Example shell script (for example in
 `/etc/archivematica/automation-tools/transfer-script.sh`):
 
 ```bash
 #!/bin/bash
 cd /usr/lib/archivematica/automation-tools/
+# Use the Python in the virtual environment.
```
Comment (ross-spencer) [inline README.md]: Could possibly consider removing the definitive article before `Python` but is still clear otherwise.

File: README.md [hunk 3]
```diff
@@ -514,8 +533,8 @@ Service using `dips.storage_service_upload` and `dips.atom_upload` or as part of
 the `aips/create_dips_job.py` execution.
 
 The AtoM upload requires a passwordless SSH connection to the AtoM host for the
-user running the script. The AtoM host must be added to list of known hosts.
-[More info][Send DIPS].
+user running the script. The AtoM host must be added to list of known hosts
+using this [guide](https://wiki.archivematica.org/Upload_DIP#Send_your_DIPs_using_rsync).
```
Comment (ross-spencer) [inline README.md]: Nice :+1:

File: etc/transfer-script.sh [hunk 4]
```diff
@@ -2,4 +2,10 @@
...
 # /etc/archivematica/automation-tools/transfer-script.sh
 cd /usr/lib/archivematica/automation-tools/
-/usr/share/python/automation-tools/bin/python -m transfers.transfer --user <user> --api-key <apikey>  --ss-user <user> --ss-api-key <apikey> --transfer-source <transfer_source_uuid> --config-file <config_file>
+/usr/share/python/venv/automation-tools/bin/python -m transfers.transfer 
+  --user <user> 
+  --api-key <apikey>  
+  --ss-user <user> 
+  --ss-api-key <apikey> 
+  --transfer-source <transfer_source_uuid> 
+  --config-file <config_file>
```
Comment (ross-spencer) [inline etc/transfer-script.sh]: Thanks for doing this!

**Comment by ross-spencer** [comment]
Nice work! Thank you @jorikvankemenade :+1: 

Looking forward to your next PRs! :wink: (How will you be using the automation-tools out of interest?)

File: etc/transfer-script.sh [hunk 5]
```diff
@@ -2,4 +2,10 @@
 # transfer script example
 # /etc/archivematica/automation-tools/transfer-script.sh
 cd /usr/lib/archivematica/automation-tools/
-/usr/share/python/automation-tools/bin/python -m transfers.transfer --user <user> --api-key <apikey>  --ss-user <user> --ss-api-key <apikey> --transfer-source <transfer_source_uuid> --config-file <config_file>
+/usr/share/python/venv/automation-tools/bin/python -m transfers.transfer
```
Comment (jraddaoui) [inline etc/transfer-script.sh]: Hi @jorikvankemenade and @ross-spencer, shouldn't this be `/usr/share/python/automation-tools/venv/bin/python`?

File: README.md [hunk 6]
```diff
@@ -53,113 +53,132 @@ Follow each of the steps below to install automation-tools:
...
-transfer tool. Example shell script (for example in
+Suggested deployment is to use cron to run a shell script that runs the 
+automation transfer tool. Example shell script (for example in
 `/etc/archivematica/automation-tools/transfer-script.sh`):
 
 ```bash
 #!/bin/bash
 cd /usr/lib/archivematica/automation-tools/
+# Use the Python in the virtual environment.
 /usr/share/python/automation-tools/bin/python -m transfers.transfer \
```
Comment (jraddaoui) [inline README.md]: @jorikvankemenade, @ross-spencer, `/usr/share/python/automation-tools/venv/bin/python` in here too? And there are other scripts in `etc` that will also need this change.

File: etc/transfer-script.sh (hunk 5 above)
Comment (ross-spencer) [reply etc/transfer-script.sh]: You are right @jraddaoui, completely missed that. 

@jorikvankemenade are you able to make the change @jraddaoui requests here and below and submit a new PR on top of the new `master` branch?

File: README.md (hunk 6 above)
Comment (jorikvankemenade) [reply README.md]: I'll make sure to check all scripts and open a PR later this week. Thanks for pointing this out.

File: README.md (hunk 1 above)
Comment (jorikvankemenade) [reply README.md]: Yest will do that.

**Comment by jorikvankemenade** [comment]
> Looking forward to your next PRs!  (How will you be using the automation-tools out of interest?)

I am currently investigating if we can use Archivematica for long-term preservation at CERN. For this investigation, I have to look into batch ingestion of many files. Hopefully, the automation tools can solve this, or are a good inspiration for some future work :).

**Comment by ross-spencer** [comment]
Very cool @jorikvankemenade. A few folks are considering similar things. If you need additional inspiration or would like to talk through ideas, then the [Google Group](https://groups.google.com/forum/#!topic/archivematica/OWnLl6T9qXY) is a great place to field those. It sounds like a great topic for the camp later this year too, cc. @ablwr.

**Pull request merged**